### ISSUE TITLE
Replicating Roberta's results on SuperGlue COPA task

### ISSUE THREAD

**Issue by dhairyadalal** [pytorch/fairseq#3051]
Hi,
I'm trying to replicate Roberta's results on the superglue task and getting poor results. I have a few questions about the results and input schemes.

First I noticed that the benchmark score in this repo (roberta.large: 94.0) is different from the SuperGlue leaderboard page (90.6) Which result should I consider as my reference?

Second which model is being used for the Superglue submission? Based on the description it looks like the robert-large-mnli is being used but in the repo the results state it's roberta-large. 

Finally, what is the input for the COPA task look like for Roberta? 

For example if you have a question, premise, and pair of alternatives:
EG:
Premise: The man broke his toe. 
Question: What was the CAUSE of this?
Alternative 1: He got a hole in his sock.
Alternative 2: He dropped a hammer on his foot.

I tried setting up input as: `<s>premise</s> <s> choice 1 </s> <s> choice 2 </s> <s>question </s>`.

For the model and training, I take the pooled output from robert-large-mnli and feed into linear layer for binary classification where 0 represent alt 1 and 1 alt 2. I fine tune the model for 3 epochs using the the default learning rate for adamw. 

This approach seems to do very poorly (getting accuracy of 69.0 on the test set after finetuning for 3 epochs using the roberta-large-mnli weights from the transformers library). Any suggestion on how to setup the input and training process to replicate the paper's results?

Thanks!
